set_toolchain.sh

```diff
@@ -124,10 +124,17 @@ export PERFORM
 #
 case "$lmi_build_type" in
     (*-*-linux*)
-        w0="$(winepath -w "$localbindir" | sed -e's/\\/\\\\/g')"
-        w1="$(winepath -w "$locallibdir" | sed -e's/\\/\\\\/g')"
-        export WINEPATH="$w0;$w1"
-        export  PERFORM="wine"
+        case "$LMI_TRIPLET" in
+            (*-*-mingw32)
+                w0="$(winepath -w "$localbindir" | sed -e's/\\/\\\\/g')"
+                w1="$(winepath -w "$locallibdir" | sed -e's/\\/\\\\/g')"
+                export WINEPATH="$w0;$w1"
+                export  PERFORM="wine"
+                ;;
+            (*-*-linux-gnu)
+                export LD_LIBRARY_PATH="$locallibdir":"$prefix/${LMI_COMPILER}_${LMI_TRIPLET}/build/ship"
```

Comment:
This should be adjusted as discussed, i.e. made more symmetric with `PATH` setting above.

Reply:
TBD in another PR (as discussed).